angular2/src/angular2TextMask.ts

```diff
@@ -33,6 +33,7 @@ export default class MaskedInputDirective {
   }
 
   writeValue(value: any) {
+    this.textMaskInputElement.update(value)
```

Comment:
Sure, but it might be fine! We have the same situation in React and it's fine because even though `update` is called twice, it ignores the second call as you can see here:

https://github.com/text-mask/text-mask/blob/master/core/src/createTextMaskInputElement.js#L60